src/Symfony/Bridge/Doctrine/Tests/Form/EventListener/MergeDoctrineCollectionListenerTest.php

```diff
@@ -22,28 +23,15 @@
 
 class MergeDoctrineCollectionListenerTest extends TestCase
 {
-    /** @var \Doctrine\Common\Collections\ArrayCollection */
-    private $collection;
-    /** @var \Symfony\Component\EventDispatcher\EventDispatcher */
-    private $dispatcher;
-    private $factory;
-    private $form;
+    private ArrayCollection $collection;
+    private EventDispatcher $dispatcher;
+    private MockObject&FormFactoryInterface $factory;
 
     protected function setUp(): void
     {
         $this->collection = new ArrayCollection(['test']);
         $this->dispatcher = new EventDispatcher();
         $this->factory = $this->createMock(FormFactoryInterface::class);
-        $this->form = $this->getBuilder()
-            ->getForm();
-    }
-
-    protected function tearDown(): void
-    {
-        $this->collection = null;
-        $this->dispatcher = null;
-        $this->factory = null;
-        $this->form = null;
     }
```

Comment:
let's stop adding those useless tearDown methods

Reply:
I check if the memory consumption was the same => 🎉 it's the same! 👏🏼 

---

But there is something a bit strange. about Framework bundle (I looked at this one because it fails)

* On Github, it consumes more than 500Mb
* locally, it consumes 244.00Mb

And locally, I have less skipped tests!